docs/components/collapse.md

```diff
@@ -0,0 +1,232 @@
+# Collapse
+
+**KCollapse** - A collapsible section.
+
+<KCollapse trigger-label="View more info">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+</KCollapse>
+
+```html
+<KCollapse trigger-label="View more info">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+</KCollapse>
+```
+
+## Props
+
+### title
+
+Use this prop to apply a title to the collapsible section.
+
+<KCollapse title="Look Mah!" trigger-label="View more info">
+  Can you see me now?
+</KCollapse>
+
+```html
+<KCollapse title="Look Mah!" trigger-label="View more info">
+  Can you see me now?
+</KCollapse>
+```
+
+### triggerAlignment
+
+You can customize the alignment of the trigger. `leading` or `trailing` (default) alignment is supported.
+If a `title` is specified, the trigger will be inline with `trailing` alignment, or displayed beneath the `title` with `leading` alignment.
+
+<KCollapse title="Look Mah!" trigger-label="What?" trigger-alignment="leading">
+  Can you see me now?
+</KCollapse>
+
+```html
+<KCollapse title="Look Mah!" trigger-label="What?" trigger-alignment="leading">
+  Can you see me now?
+</KCollapse>
+```
+
+### triggerLabel
+
+Use this prop to customize the content to that will toggle the collapsed state of the component. The label will be displayed to the right of a caret that indicates the state of the hidden content.
+If no label is provided, only a caret will be displayed.
+
+<KCollapse trigger-alignment="leading">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+</KCollapse>
+
+```html
+<KCollapse trigger-alignment="leading">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+</KCollapse>
+```
```

Comment:
I don't think this is the right example for this prop? It doesn't even have the `triggerLabel` prop..?

Reply:
Fixed

![image](https://user-images.githubusercontent.com/67973710/195871741-279a822c-dd96-41eb-8994-9c4194583686.png)